charts/gardener-extension-runtime-gvisor/templates/deployment.yaml

```diff
@@ -28,7 +28,6 @@ spec:
         command:
         - /gardener-extension-runtime-gvisor
         - --max-concurrent-reconciles={{ .Values.controllers.concurrentSyncs }}
-        - --disable-controllers={{ .Values.disableControllers | join "," }}
```

Comment:
Why did you remove it?

Reply:
I wasn't sure for the usage of this. do we really need this?
and i do get: {"level":"error","ts":"2020-03-22T16:04:48.618Z","msg":"error executing the main controller command","error":"unknown flag: --disable-controllers"

Reply:
OK, I guess it doesn't make any sense anyways at this point in time as there is only one controller in it, and disabling it would not bring any benefits. Thanks.